dotCMS/src/main/java/com/dotmarketing/db/DBPropertiesDatasourceStrategy.java

```diff
@@ -0,0 +1,86 @@
+package com.dotmarketing.db;
+
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Constants;
+import com.dotmarketing.util.Logger;
+import java.io.File;
+import java.net.URL;
+import javax.sql.DataSource;
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+
+/**
+ * Singleton class that provides a datasource using a <b>db.properties</b> file configuration
+ * @author nollymar
+ */
+public class DBPropertiesDatasourceStrategy implements DotDatasourceStrategy {
+
+    private final String DB_PROPERTIES_FILE_NAME = "db.properties";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This final field could be made static](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5026767)